### PR TITLE
Removed Lint errors and Cleaned Code

### LINKED ISSUE #805: Frontend: Remove lint errors and clean the code
Steps to follow:

1. Run `gulp lint` and see the errors in the terminal
2. Remove all the warnings and errors.
3. `jshint` has been removed from linting purpose so if you see any comment named `jshint` then remove it.
4. if you see warning `Unexpected console statement` then remove the console from the js file. And if console is important to be written then refer this [link](http://eslint.org/docs/rules/no-console) and it's your call that how you gonna fix that.

PS: It's and important task and need to be done ASAP.

### PR DESCRIPTION
**Pull request by jainarunarunjain**
fixes #805  @spyshiv @aka-jain Please Review The PR.

### CODE REVIEW

**Comment by spyshiv** [comment]
@jainarunarunjain : LGTM :) 
Merge/rebase master branch and then please remove conflicts. and push the code again.

File: .eslintrc [hunk 1]
```diff
@@ -11,7 +11,8 @@ eslint config file.
 	"globals": {
 		"angular":1,
 		"jquery": 1,
-		"$": 1
+		"$": 1,
+		"_": true,
```
Comment (spyshiv) [inline .eslintrc]: write `1` instead of `true`

Comment (jainarunarunjain) [inline .eslintrc]: @spyshiv sure.

**Comment by jainarunarunjain** [comment]
@spyshiv @aka-jain Done, Please review.

**Pull request merged**